Ginger/GingerCore/Actions/ActGenElement.cs

```diff
@@ -166,6 +166,7 @@ Act IObsoleteAction.GetNewAction()
                     case eGenElementAction.Focus:
                         NewActUIElement.ElementAction = ActUIElement.eElementAction.SetFocus;
                         break;
+                    case eGenElementAction.SetAttributeUsingJs:
                     case eGenElementAction.RunJavaScript:
```

Comment:
we can't convert SetAttributeUsingJs to eElementAction.RunJavaScript because it has diffrent implementation, it won't pull the attribute value with UIElementAction.
leave it, no need to convert it.